python/ingestion/standardized_columns.py

```diff
@@ -127,6 +127,15 @@
 HIV_DIAGNOSES_PER_100K = 'hiv_diagnoses_per_100k'
 HIV_DIAGNOSES_PCT_INEQUITY = "hiv_diagnoses_pct_relative_inequity"
 HIV_DIAGNOSES_PCT_SHARE = 'hiv_diagnoses_pct_share'
+HIV_DEATHS = 'hiv_deaths'
+HIV_DEATHS_PER_100K = 'hiv_deaths_per_100k'
+HIV_DEATHS_PCT_INEQUITY = "hiv_deaths_pct_relative_inequity"
+HIV_DEATHS_PCT_SHARE = 'hiv_deaths_pct_share'
+HIV_PREP = 'hiv_prep'
+HIV_PREP_COVERAGE = 'hiv_prep_coverage'
+HIV_PREP_PCT_INEQUITY = 'hiv_prep_pct_relative_inequity'
+HIV_PREP_PCT_SHARE = 'hiv_prep_pct_share'
+HIV_PREP_POPULATION = 'hiv_prep_population'
```

Comment:
maybe worth adding a comment here explaining why there are unique population columns for PREP vs other HIV indicators?